src/NBomber/Api/FSharp.fs

```diff
@@ -45,7 +48,9 @@ module Scenario =
     let withDuration (duration: TimeSpan) (scenario: Scenario) =
         { scenario with Duration = duration }
 
-    let addAssertions(assertions: Assertion list) (scenario: Scenario) =
-        { scenario with Assertions = assertions |> Seq.toArray }
+    let run (scenario: Scenario) = Run(scenario, true) |> ignore
+
+    let runWithAssertions (scenario: Scenario, assertions: IAssertion []) =
```

Comment:
let runWithAssertions (assertions: IAssertion []) (scenario: Scenario) =
now you can write
|> Scenario.runWithAssertions(assertions)